This file is part of MXE.
See index.html for further information.

Remove an annoying runtime warning that pops up when using GtkApplication
in Gtk+ 3 programs.

--- a/gio/gdbusaddress.c	2014-06-28 19:02:43.000000000 +0200
+++ b/gio/gdbusaddress.c	2015-08-12 20:26:45.931228430 +0200
@@ -1325,6 +1325,7 @@
 __declspec(dllexport) void CALLBACK
 g_win32_run_session_bus (HWND hwnd, HINSTANCE hinst, char *cmdline, int nCmdShow)
 {
+  /*
   GDBusDaemon *daemon;
   GMainLoop *loop;
   const char *address;
@@ -1354,6 +1355,7 @@
 
   g_main_loop_unref (loop);
   g_object_unref (daemon);
+  */
 }
 
 static gchar *
